examples/kuzu/Papers-100M/README.md

```diff
@@ -0,0 +1,18 @@
+# Papers-100M example
+
+This example shows how to use the remote backend feature of Kùzu to work with a large graph of papers and citations on a single machine. The data used in this example is `ogbn-papers100M` from the [Open Graph Benchmark](https://ogb.stanford.edu/). The dataset contains approximately 100 million nodes and 2.5 billion edges.
+
+## Install the dependencies
+- numpy
+- pandas
+- kuzu
+- tqdm
+
+## Prepare the data
+
+1. Download the dataset from [http://snap.stanford.edu/ogb/data/nodeproppred/papers100M-bin.zip](http://snap.stanford.edu/ogb/data/nodeproppred/papers100M-bin.zip). Put the zip file into this directory.
```

Comment:
Can't we use OGB directly here without manual download?

Reply:
Does ogb provides a way of downloading the data without loading it into the RAM? When I called `dataset = PygNodePropPredDataset(name='ogbn-papers100M')`, it downloads the data and immediately starts to load and convert the data with a peak RAM usage of ~120 GB. This seem to defeat the purpose of using remote backend, as it would crash on systems without that much RAM.

Reply:
I see, I was somehow mistaken that OGB provides an interface to download without loading it into RAM :(